{{ cookiecutter.github_repo_name }}/.github/ISSUE_TEMPLATE/release_checklist.md

```diff
@@ -13,30 +13,30 @@ assignees: ""
 - [ ] License information is verified as correct. If you are unsure, please comment below.
 - [ ] Locally rendered documentation contains all appropriate pages, including API references (check no modules are
       missing), tutorials, and other human-written text is up-to-date with any changes in the code.
-- [ ] Installation instructions in the README, documentation, and the website (e.g., diffpy.org) are updated.
+- [ ] Installation instructions in the README, documentation, and the website are updated.
 - [ ] Successfully run any tutorial examples or do functional testing with the latest Python version.
 - [ ] Grammar and writing quality are checked (no typos).
 - [ ] Install `pip install build twine`, run `python -m build` and `twine check dist/*` to ensure that the package can be built and is correctly formatted for PyPI release.
 
-Please mention @sbillinge here when you are ready for PyPI/GitHub release. Include any additional comments necessary, such as version information and details about the pre-release here:
+Please tag the maintainer when you are ready for PyPI/GitHub release. Include any additional comments necessary, such as version information and details about the pre-release here:
```

Comment:
I guess in principle we know the maintainer gh username, so we could template some or all of these mentions of maintainer if we thought it added value.  Or we could leave it as maintainer.  Or to cover all bases we could be super-clever and do `.. a maintainer (e.g., @<maintainer-name>) when..

Reply:
@sbillinge I tried... It turns out cookiecutter can't recognize {{ cookiecutter.maintainer... }} within files under `.github` while it works for hidden files that exist at the top level like `.isort`...

Reply:
Perhaps cookiecutter isn't trying to modify anything under folders like `.git`, etc. to avoid any serious problems.. on purpose.